hardware/fpga/alsaqr/tcl/ips/clk_mngr/run.tcl

```diff
@@ -10,7 +10,11 @@ if [info exists ::env(BOARD)] {
 if {$::env(MAIN_MEM)=="HYPER"} {
     set CLK_FREQ_MHZ 10
 } elseif {$::env(MAIN_MEM)=="DDR4"} {
-    set CLK_FREQ_MHZ 40
+    if {$::env(NUM_CORES)==4} {
+        set CLK_FREQ_MHZ 20
```

Comment:
Why we go so slow when NUM_CORES is 4?

Reply:
more utilization, higher implementation time, slower frequency